console_other.go

```diff
@@ -0,0 +1,35 @@
+// +build !darwin,!freebsd,!linux,!netbsd,!openbsd,!solaris
```

Comment:
Please gofmt with Go 1.17

Reply:
@AkihiroSuda If I do that, the license checker will fail. I previously sent the PR with it formatted that way, and the test failed. The containerd license checker expects this format and doesn't allow the other one.

Reply:
```
    - name: Check file headers
      run: ../project/script/validate/fileheader ../project/
      working-directory: src/github.com/containerd/console

```